### ISSUE TITLE
Unable to set customfield from JiraPS using Powershell

### ISSUE THREAD

**Issue by tonyitde** [AtlassianPS/JiraPS#422]
Dear community,

I am new here and it is my first time trying to use JiraPS for automation. I am trying to update a service request changing a value we have Request type used by customfield_10000

![image](https://user-images.githubusercontent.com/68995806/88928926-b05d4600-d279-11ea-89a9-2b8e7a95d327.png)

I want to update the value from this field using Set-JiraIssue but I am not sure how can I do that. I checked the metadata

![image](https://user-images.githubusercontent.com/68995806/88928193-ce767680-d278-11ea-915b-b10e4fa334e3.png)

![image](https://user-images.githubusercontent.com/68995806/88928508-3331d100-d279-11ea-8d49-cf70557f337d.png)

I want to update a ticket from one Request type to another, for example, using the last image from the ID 23 to other ID from the value list allowed.

then I tried multiple combinations using Fields

> $Fields = @{}
> $Fields.Add('customfield_10000' , NO IDEA)

can you please help me using JiraPS? Reading a ticket from Powershell I discover the following for customfield_10000

![image](https://user-images.githubusercontent.com/68995806/88929197-14800a00-d27a-11ea-8900-8258a62eb483.png)

Thanks a lot for your kindly suggestions
Tony

**Comment by jdelforno** [comment]
@tonyitde Post your code?

**Comment by lipkau** [comment]
hard to help without code.
But considering the schema of the field, I would start with 

```powershell
Set-JiraIssue $issue -Fields @{ customfield_10000 = @{ id = 22 } }
```

**Comment by tonyitde** [comment]
Dear Team,

sorry I was really off of IT topic last weeks. 

The solution came to me finally: using F12 I send in the web portal different request types. I analyzed what was sent and I discover the field customfield_10000 sent a long string, changing for each different type. I created than using JiraPS a ticket adding this value and works!

This is a plugin then if more of you use it, you have the change to create new ticket with populated value.

Regards Tony
